src/components/Carousel/Carousel.js

```diff
@@ -1,14 +1,18 @@
 import { useState } from 'react';
+import Swipe from 'react-easy-swipe';
 import './Carousel.css';
 import Button from '../Button/Button';
 import { data, arrows } from '../../config/products';
 
 export default function Carousel(props) {
   const [currentIndex, setCurrentIndex] = useState(0);
 
-  const handleArrowClick = (evt) => {
-    const isLeftArrow = evt.currentTarget.classList.contains('carousel-arrow_left');
-    const newIndex = currentIndex + (isLeftArrow ? -1 : 1);
+  const handleArrowClick = (evt, isLeft) => {
+    if (evt.currentTarget) {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
I'm tempted to just disable this lint rule entirely, it is a pain. I can't really think of a good pattern to handle this otherwise.